### PR TITLE
merge develop into master

### PR DESCRIPTION
**Pull request by LorenzoGaetaniFinc**
Added service controller mapper for Lane

### CODE REVIEW

File: src/main/java/com/fincons/taskmanager/service/laneService/impl/LaneServiceImpl.java
Comment (Simone-Puce) [inline src/main/java/com/fincons/taskmanager/service/laneService/impl/LaneServiceImpl.java:1]: Why is the package name capital

File: src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java [hunk 1]
```diff
@@ -51,11 +49,11 @@ public Task updateTaskByCode(String taskCode, Task task) {
         List<Task> tasks = taskRepository.findAll();
         Task taskExisting = validateTaskByCode(taskCode);
 
-        List<Task> tasksWithoutTaskCodeChosed = new ArrayList<>();
+        List<Task> tasksWithoutTaskCodeChosen = new ArrayList<>();
```
Comment (Simone-Puce) [inline src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java]: what is this method supposed to do

**Review by Simone-Puce — CHANGES_REQUESTED** [review]

File: src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java (hunk 1 above)
Comment (LorenzoGaetaniFinc) [reply src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java]: This is a put method

Comment (Simone-Puce) [reply src/main/java/com/fincons/taskmanager/service/taskService/impl/TaskServiceImpl.java]: tasksExcludingSelectedTask use this name

**Review by Simone-Puce — APPROVED** [review]
approved

**Pull request merged**